tests/integration/security/pass_through_filter_chain_test.go

```diff
@@ -115,7 +115,7 @@ func TestPassThroughFilterChain(t *testing.T) {
 					},
 
 					// For workload c, there is only authN policy that enables mTLS (Strict).
-					// The request should be denied because the x is always using plain text.
+					// The request should be denied because the e is always using plain text.
```

Comment:
Is this change intended (replacing x with e)?

Reply:
Yes